code/backend/batch/utilities/helpers/AzureSearchIVSkillsetHelper.py

```diff
@@ -0,0 +1,103 @@
+import logging
+from azure.search.documents.indexes.models import (
+    SplitSkill,
+    InputFieldMappingEntry,
+    OutputFieldMappingEntry,
+    AzureOpenAIEmbeddingSkill,
+    SearchIndexerIndexProjections,
+    SearchIndexerIndexProjectionSelector,
+    SearchIndexerIndexProjectionsParameters,
+    IndexProjectionMode,
+    SearchIndexerSkillset,
+)
+from azure.search.documents.indexes import SearchIndexerClient
+from .EnvHelper import EnvHelper
+from azure.identity import DefaultAzureCredential
+from azure.core.credentials import AzureKeyCredential
+
+logger = logging.getLogger(__name__)
+
+
+class AzureSearchIVSkillsetHelper:
+    def __init__(self, env_helper: EnvHelper):
+        self.env_helper = env_helper
+        self.indexer_client = SearchIndexerClient(
+            self.env_helper.AZURE_SEARCH_SERVICE,
+            (
+                AzureKeyCredential(self.env_helper.AZURE_SEARCH_KEY)
+                if self.env_helper.AZURE_AUTH_TYPE == "keys"
+                else DefaultAzureCredential()
+            ),
+        )
+
+    def create_skillset(self):
+        skillset_name = f"{self.env_helper.AZURE_SEARCH_INDEX}-skillset"
+
+        split_skill = SplitSkill(
+            description="Split skill to chunk documents",
+            text_split_mode="pages",
+            context="/document",
+            maximum_page_length=2000,
+            page_overlap_length=500,
```

Comment:
Can these sizes be made configurable? If we have an existing chunk size configuration, can we reuse them here?